MapboxVision/VisionManager/VisionManager.swift

```diff
@@ -98,7 +122,9 @@ public final class VisionManager: BaseVisionManager {
     }
 
     /**
-     Cleanup the state and resources of `VisionManger`.
+     Clean up the state and resources of `VisionManger`.
+
+     - Important: Do NOT call this method more than once.
```

Comment:
Should this method be called before destructor?

Reply:
It may but if you you need to clean up resources earlier but it does not have to since it's called in `deinit`. This message is here to motivate the expected lifecycle.